src/v0/sources/shopify/transform.js

```diff
@@ -177,7 +198,8 @@ const isIdentifierEvent = (event) => {
   }
   return false;
 };
-const processIdentifierEvent = () => {
+const processIdentifierEvent = async (event) => {
+  await redisConnector.postToDB(event.cart.token, JSON.stringify(event));
```

Comment:
Let's add a try-catch block to catch errors.

Reply:
Done